perf/sds-tests/non-sds-cert-rotation/run-test.sh

```diff
@@ -0,0 +1,47 @@
+# Create a cluster for testing
+# Set PROJECT as the name of your GCP project
+export PROJECT="endpoints-jenkins"
+# Set CLUSTER as the name of your newly created GKE cluster 
+export CLUSTER="istio-cluster-test-sds-disabled"
+# Set ZONE as the zone of your newly created GKE cluster 
+export ZONE="us-central1-a"
+# Set RELEASE_NAME as the release name you are going to test
+export RELEASE_NAME=release-1.1-20190129-09-16
+
+gcloud container clusters get-credentials ${CLUSTER} --zone ${ZONE} --project ${PROJECT} 
+
+kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user="$(gcloud config get-value core/account)"
+
+wget https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/$RELEASE_NAME/istio-${RELEASE_NAME}-linux.tar.gz
+
+#TODO: may need to delete previously downloaded release, if any 
+tar xfz istio-${RELEASE_NAME}-linux.tar.gz
```

Comment:
Don’t copy so much code. It is already there. We can’t maintain 2 copies.

Reply:
This file has been removed.